services/motion/builtin/builtin_test.go

```diff
@@ -652,3 +677,200 @@ func TestGetPose(t *testing.T) {
 	test.That(t, err, test.ShouldBeError, referenceframe.NewParentFrameMissingError("testFrame", "noParent"))
 	test.That(t, pose, test.ShouldBeNil)
 }
+
+func TestStoppableMoveFunctions(t *testing.T) {
+	ctx := context.Background()
+	logger := golog.NewTestLogger(t)
+	failToReachGoalError := errors.New("failed to reach goal")
+	calledStopFunc := false
```

Comment:
I think this line isn't necessary

Reply:
Do you mean because `calledStopFunc` could be (and partially is already) defined within each `t.Run`?

Reply:
Yeah that was my reading, but either way its fine

Reply:
Turns out it's necessary for the function definition right after.